plugins/org.yakindu.sct.model.stext/src/org/yakindu/sct/model/stext/validation/STextJavaValidator.java

```diff
@@ -199,7 +203,63 @@ public void transitionsWithNoTrigger(Transition trans) {
 			warning(ISSUE_TRANSITION_WITHOUT_TRIGGER, trans, null, -1);
 		}
 	}
+	
+	@Check(CheckType.FAST)
+	public void checkAlwaysTransitionHasLowestPriority(RegularState state) {
```

Comment:
Also add warnings for all transitions that have lower priority. Use message like:

' Dead transition. This transition can not be taken due to previous transition with '%s' trigger.